lib/rubocop/cop/betterment/no_strict_loading.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Betterment
+      module NoStrictLoading
```

Comment:
I'm never really sure how to name cops, but this one did feel a little confusing to me. We _want_ strict loading, but this name makes it seem like using strict loading is a bad thing.

Reply:
__comment:__ It's more that we don't want the code to specify strict loading; we want to rely on the global settings, with the global setting turned on.

Adding an explicit strict loading config (both on or off) means you're overriding the global config, which I think is a code smell.

Reply:
__comment:__ I'll try out `UseGlobalStrictLoading` as a namespace since it matches my intention more closely.